internal/gpustats/nvidia_test.go

```diff
@@ -138,3 +138,7 @@ func TestNvidiaSmiInvalidDataParse(t *testing.T) {
 		}
 	}
 }
+
+func TestNvidiaCorrectNamePopulation(t *testing.T) {
+
```

Comment:
It looks like you wanted to have something here??

Reply:
Had something in mind, then promptly forgot about it. I really should have looked over this pr more before i requested review

Reply:
I think you need some sort of test for PopulateNames here.